SIS_import/.env_example

```diff
@@ -3,8 +3,10 @@ canvas_allowed_call_number=2900
 alert_email_address=EMAIL_ADDRESS
 
 # for sis_upload
+current_directory=PATH
 sleep=20
 practice_course_subaccount=ACCOUNT_ID
+aws_s3_bucket=BUCKET_NAME
```

Comment:
Normally, convention is that [environment variables are all caps](https://stackoverflow.com/a/673940/3708872) so they stand out more in the code. Though this is fine for this value since it's already the pattern here.

Reply:
I've changed all env variables into caps. Please check the latest code.